content/docs/02.installation/14.podman-compose.md

```diff
@@ -51,3 +51,32 @@ The [configuration](/docs/configuration-guide) will be done inside the `KESTRA_C
 ::alert{type="info"}
 If you want to extend your Docker Compose file, modify container networking, or if you have any other issues using this Docker Compose file, check the [Troubleshooting Guide](/docs/faq/troubleshooting).
 ::
+
+### Use a configuration file
+
+If you want to use a configuration file instead of the `KESTRA_CONFIGURATION` environment variable to configure Kestra you can update the default `docker-compose.yml`.
+
+First, create a configuration file, for example named `application.yaml` and put inside the content of the `KESTRA_CONFIGURATION` environment variable defined in the `docker-compose.yml` file.
+
+Then update `kestra` servce in the `docker-compose.yml` file to mount this file into the container and make Kestra using it via the `--config` option:
+
+```yaml
+# [...]
+  kestra:
+    image: kestra/kestra:latest-full
+    pull_policy: always
+    # Note that this is meant for development only. Refer to the documentation for production deployments of Kestra which runs without a root user.
+    user: "root"
+    command: server standalone --worker-thread=128 --config /etc/config/application.yaml
+    volumes:
+      - kestra-data:/app/storage
+      - /var/run/docker.sock:/var/run/docker.sock
+      - /tmp/kestra-wd:/tmp/kestra-wd
+      - $PWD/application.yaml:/etc/config/application.yaml
+    ports:
+      - "8080:8080"
+      - "8081:8081"
+    depends_on:
+      postgres:
+        condition: service_started
+```
```

Comment:
can we just reference the section instead of duplicating the content?

Reply:
As some lines of the service definition are removed, some added and some modified I found it easier to include the full service definition (but not the full docker compose file) so a user can copy/paste it and replace it;